packages/apps/desktop-viewer-test/src/common/LoggerCategory.ts

```diff
@@ -0,0 +1,10 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+/** List of LoggerCategories for this app.  For more details on Logging Categories, check out the [Category](https://www.imodeljs.org/learning/common/logging/#categories) documentation. */
+export enum AppLoggerCategory {
```

Comment:
How does logging work in a desktop app in production? Like are we writing out to a file or sending logs back to seq or something else via OpenTelemetry under the hood?

Reply:
in this app we are initializing to the console, but the logger class has different options that users could use to provide metadata, etc.